recipes/dry-attr/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.3.2" %}
+{% set name = "dry-attr" %}
```

Comment:
Why have you prefixed this with dry?

Reply:
👍 Normally, we would use `python-attr` as a way to avoid this conflict. It probably makes sense to add that alias long-term.